crypto/conf/conf.c

```diff
@@ -642,6 +642,18 @@ int CONF_modules_load_file(const char *filename, const char *appname,
   return 1;
 }
 
+char *CONF_get1_default_config_file(void) {
+  const char *temp = "No support for Config files in AWS-LC.";
+  size_t temp_len = strlen(temp) + 1;
+
+  char *ret = (char *)OPENSSL_malloc(temp_len);
```

Comment:
note for other reviewers: OpenSSL [also](https://github.com/openssl/openssl/blob/0285160ffa3b8c2b5491222243042593808298c4/crypto/conf/conf_mod.c#L698) alloc's in this method and leaves `free`ing up to the caller.